bigchaindb/models.py

```diff
@@ -118,8 +118,36 @@ def validate(self, bigchain):
 
 
 class Block(object):
+    """A Block bundles up to 1000 Transactions. Nodes vote on its validity.
+
+    Attributes:
+        transaction (:obj:`list` of :class:`~.Transaction`):
+            Transactions to be included in the Block.
+        node_pubkey (str): The public key of the node creating the
+            Block.
+        timestamp (str): The Unix time a Block was created.
+        voters (:obj:`list` of :obj:`str`): A list of a federation
+            nodes' public keys supposed to vote on the Block.
+        signature (str): A cryptographic signature ensuring the
+            integrity and creatorship of a Block.
```

Comment:
not sure if `creatorship` is the word we wanna use here. I'd say `A cryptographic signature ensuring the integrity and authenticity of a Block.`